docs/integrations/role_mapping.md

```diff
@@ -0,0 +1,87 @@
+# IAM role mapping
+
+Using an EC2 instance profile enables your Hopsworks cluster to access AWS resources. This forces all Hopsworks users to
+share the instance profile role and the resource access policies attached to that role. To allow for per project access policies
+you could have your users use AWS credentials directly in their programs which is not recommended so you should instead use
+[Role chaining](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#iam-term-role-chaining).
```

Comment:
(also known as [credential passthrough[(https://docs.databricks.com/security/credential-passthrough/iam-passthrough.html)).

Reply:
Is it wise to reference databricks here?

Reply:
We are so small, i don't think it really matters (we really matter)!. But fine, leave it out.